### DISCUSSION TITLE
Edge Middleware not deploying with Remix

### DISCUSSION THREAD

**Discussion by willhighland** [vercel/community#2205] [Bug Report]
I created a [simple repo](https://github.com/willhighland/remix-boilerplate) using the `remix-boilerplate` [template](https://vercel.com/templates/remix/remix-boilerplate) on the marketplace. I then added a `middleware.js` file to the root of the project, following [the instructions here](https://vercel.com/docs/concepts/functions/edge-middleware/quickstart#create-edge-middleware-with-vercel-cli). The middleware should redirect from `/blocked` to `/index`. However, it doesn't appear as if the middleware is being deployed correctly and I don't see an entry for `Edge Middleware` in my deployment summary. (see screenshot below) 
<img width="1236" alt="CleanShot 2023-04-26 at 13 44 54@2x" src="https://user-images.githubusercontent.com/2426048/234685830-50e9404c-747b-4a01-a915-564f6fc9dad9.png">

I have tried following the same steps for a non-remix project and I was able to see the edge middleware deployed and functioning correctly. I believe this may be an issue specific to the remix integration that is preventing middleware from deploying and executing correctly.

Link to repo: https://github.com/willhighland/remix-boilerplate

**Comment by EndangeredMassa** [comment]
We're investigating and will get back to you about this.

**Comment by sebastianroming** [comment]
@willhighland We've released a fix for this and Edge Middleware should now be correctly applied. Can you try redeploying your project?

**Comment by willhighland** [comment]
Hi @sebastianroming, this looks to be working now. Thank you for the help!
